dev-deployment/docker-compose.yml

```diff
@@ -0,0 +1,199 @@
+# This file is prepared only for dev purposes
+# It does not contain depends statement for registry microservice
+# Do not use it as compose descriptor on the production.
+version: '2.4'
+services:
+
+  postgres:
+    image: postgres:12-alpine
+    environment:
+      POSTGRES_USER: rpuser
+      POSTGRES_PASSWORD: rppass
+      POSTGRES_DB: reportportal
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
+    restart: on-failure
+    # If you need to access the DB locally. Could be a security risk to expose DB.
+    ports:
+      - "5432:5432"
+
+  rabbitmq:
+    image: rabbitmq:3.7.16-management
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    environment:
+      RABBITMQ_DEFAULT_USER: "rabbitmq"
+      RABBITMQ_DEFAULT_PASS: "rabbitmq"
+
+  gateway:
+    image: traefik:v2.0.5
+    ports:
+      - "8080:8080" # HTTP exposed
+      - "8081:8081" # HTTP Administration exposed
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    command:
+      - --providers.docker=true
+      - --providers.docker.constraints=Label(`traefik.expose`, `true`)
+      - --entrypoints.web.address=:8080
+      - --entrypoints.traefik.address=:8081
+      - --api.dashboard=true
+      - --api.insecure=true
+    restart: always
+
+  migrations:
```

Comment:
Please adjust migrations timeout since the first time I run the script it failed with timeout.

Reply:
done